oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/authentication/OAuth2AccessTokenAuthenticationToken.java

```diff
@@ -38,9 +38,10 @@
  */
 public class OAuth2AccessTokenAuthenticationToken extends AbstractAuthenticationToken {
 	private static final long serialVersionUID = SpringSecurityCoreVersion2.SERIAL_VERSION_UID;
-	private final RegisteredClient registeredClient;
-	private final Authentication clientPrincipal;
 	private final OAuth2AccessToken accessToken;
+	private RegisteredClient registeredClient;
+	private Authentication clientPrincipal;
+	private String clientId;
```

Comment:
`clientId` is not used. I think this was left over from previous updates? Please revert the changes in this file.